src/pages/CategoriesPage.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+const Categories = () => (
+  <div>
+    <div><p /></div>
+    <div><p /></div>
+    <button type="button">Check status</button>
+    <p>HI</p>
```

Comment:
- [ ] kindly remove this 'Hi' message as previously requested, the categories page should display only `check status` button.

Reply:
Done, just forgot to delete it, However, I don't see it as a cause to not approve the project